src/main/java/io/zold/api/Wallet.java

```diff
@@ -122,6 +130,17 @@ public Fake(final long id, final Transaction... transactions) {
             this(id, new IterableOf<>(transactions));
         }
 
+        /**
+         * Complete Ctor.
+         * @param id The wallet id.
+         * @param pubkey The wallet public key.
+         * @param network The network the walet belongs to.
+         * @checkstyle UnusedFormalParameter (2 lines)
+         */
+        public Fake(final long id, final String pubkey, final String network) {
```

Comment:
@paulodamaso I'm not sure what you intend `pubkey` to be. Take a look at [this](https://github.com/zold-io/java-api#decisions-and-alternatives).

Reply:
@llorllale As defined in [zold whitepaper](https://papers.zold.io/wp.pdf) page 9, wallets are composed of:
> 1. Network name, [a-z]{4,16} ;
> 2. Protocol version, [0-9]+ ;
> 3. Wallet ID, a 64-bit unsigned integer in hexadecimal format;
> 4. Public RSA key of the wallet owner, in Base64.
So, `pubkey` refers to the public RSA of the wallet owner.

Reply:
@paulodamaso well, yes. My bad - I was not specific enough.

My concern was with the level of abstraction. We're using the `java.security` package, so this RSA public key could perhaps be better expressed as a parameter with type [RSAPublicKey](https://docs.oracle.com/javase/8/docs/api/java/security/interfaces/RSAPublicKey.html).